vendor/modules.txt

```diff
@@ -452,7 +452,7 @@ github.com/modern-go/reflect2
 github.com/mwitkow/go-conntrack
 # github.com/ncabatoff/go-seq v0.0.0-20180805175032-b08ef85ed833
 github.com/ncabatoff/go-seq/seq
-# github.com/ncabatoff/process-exporter v0.0.0-00010101000000-000000000000 => github.com/grafana/process-exporter v0.7.3-0.20200902205007-6343dc1182cf
+# github.com/ncabatoff/process-exporter v0.7.5 => github.com/grafana/process-exporter v0.7.3-0.20210106202358-831154072e2a
```

Comment:
Unfamiliar with this process, should we version our fork to 7.5?

Reply:
We haven't been adding new tags on our forks to date,  which has the unfortunate effect of the Go modules always showing that we're one patch version above the latest upstream tag at the time of forking.

But this is actually based off of the v0.7.5 code if you compare the trees